scipy/stats/morestats.py

```diff
@@ -2202,8 +2277,13 @@ def ansari(x, y):
         else:  # N even
             varAB = m * n * (16*fac - N*(N+2)**2) / (16.0 * N * (N-1))
 
-    z = (AB - mnAB) / sqrt(varAB)
-    pval = distributions.norm.sf(abs(z)) * 2.0
+    z = (mnAB - AB) / sqrt(varAB)
```

Comment:
Does it make sense to use the function you wrote for the other tests (when that is merged tomorrow)?

Reply:
Yes, it will be better to use that function. I will merge master or rebase once you merge that PR.

Reply:
I would wait to fix this until the end if you were going to rebase. I'd like to preserve the history while we're reviewing.

Reply:
> I'd like to preserve the history while we're reviewing.

Ah, OK. I will merge master then.

Reply:
Sorry it took so long to merge gh-13549, but I think it was good to revisit it. 

Just a reminder that after you merge master you can use that nice function you wrote.  The discussion there might also affect how we want to describe the null/alternative hypotheses here.

No rush on this. Whenever you're happy with it, please ping me and I'll back to take a look.